templates/download/alternative-downloads.html

```diff
@@ -1,169 +1,181 @@
+        <li class="p-list__item"><a class="p-link--external" href="http://ports.ubuntu.com/ubuntu-ports/dists/trusty-updates/main/installer-arm64/current/images/generic/netboot/xgene/">m400 ARMv8 netboot image (14.04 LTS)</a></li>
+        <li class="p-list__item"><a class="p-link--external" href="http://h20564.www2.hp.com/hpsc/doc/public/display?docId=c03933547">HP ProLiant Moonshot user guide</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <div class="u-align--center u-vertically-center u-hidden--small" style="min-height: 8rem;">
+        <img src="{{ ASSET_SERVER_URL }}4fc650f0-Marvell_Logo.svg" alt="" width="150" />
+      </div>
+      <h3 class="p-card__title">Marvell Armada XP</h3>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a class="p-link--external" href="http://ports.ubuntu.com/ubuntu-ports/dists/precise-updates/main/installer-armhf/current/images/armadaxp/netboot">12.04 LTS</a></li>
+        <li class="p-list__item"><a class="p-link--external" href="https://wiki.ubuntu.com/ARM/Server/Install/ArmadaXP">Installation instructions for Armada XP-based systems</a></li>
+      </ul>
+    </div>
+    <div class="col-6 p-card">
+      <div class="u-align--center u-vertically-center u-hidden--small" style="min-height: 8rem;">
+        <img src="{{ ASSET_SERVER_URL }}43b35b55-logo-cavium.png" alt="" width="150" />
+      </div>
+      <h3 class="p-card__title">Cavium Thunder 48 and 96 core ARMv8</h3>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a class="p-link--external" href="http://cavium.com/ThunderX_ARM_Processors.html">Cavium ThunderX product details </a></li>
+        <li class="p-list__item"><a class="p-link--external" href="http://www.cavium.com/ContactCavium.htm">14.04 LTS Tech Preview available directly from Cavium</a></li>
+      </ul>
     </div>
+  </div>
 </div>
 
-<div class="row no-border">
-    <div class="strip-inner-wrapper">
-        <div class="eight-col">
-            <h2>Past releases and other flavours</h2>
-            <p>Looking for an older release of Ubuntu? Whether you need an obsolete release or a previous LTS point release with its original stack, you can find them in past releases.</p>
-            <p><a class="external download-pastreleases" href="http://releases.ubuntu.com/">Past releases</a></p>
-        </div><!-- /.eight-col -->
-    </div><!-- /.strip-inner-wrapper -->
+<div class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Past releases and other flavours</h2>
+      <p>Looking for an older release of Ubuntu? Whether you need an obsolete release or a previous LTS point release with its original stack, you can find them in past releases.</p>
+      <p><a class="p-link--external" href="http://releases.ubuntu.com/">Past releases</a></p>
+    </div><!-- /.col-8 -->
```

Comment:
I've been removing these comments are most modern IDE's have element collapse built in making them redundant - they also get outdated really easily.